src/notation/internal/engravingfontscontroller.cpp

```diff
@@ -39,17 +40,30 @@ std::string EngravingFontsController::moduleName() const
 
 void EngravingFontsController::init()
 {
-    mu::engraving::Smufl::init();
+    // Standard locations as described in https://w3c.github.io/smufl/latest/specification/font-metadata-locations.html
+    QStringList systemFontsPaths = QStandardPaths::standardLocations(QStandardPaths::GenericDataLocation).first(2);
+
+#ifdef Q_OS_WIN
+    // On Windows, the second standard location is C:/ProgramData, but we want C:/Program Files/Common Files
```

Comment:
No, we don't, The SMuFL doc seems wrong here, the location really is`%COMMONPROGRAMFILES%`, which does resolves to `C:/ProgramData`, at least on Windows 11. I know, because I tried...
In Mu3 I have:
```cpp
      // as per https://w3c.github.io/smufl/latest/specification/font-metadata-locations.html
      // Window: "%LOCALAPPDATA%/SMuFL/Fonts", "%COMMONPROGRAMFILES%/SMuFL/Fonts"
      // Mac:    "~/Library/Application Support/SMuFL/Fonts", "/Library/Application Support/SMuFL/Fonts"
      // Linux:  "$XDG_DATA_HOME/SMuFL/Fonts", "$XDG_DATA_DIRS/SMuFL/Fonts"
      // as per https://doc.qt.io/qt-5/qstandardpaths.html#standardLocations that is the (start of the) list
      // which `GenericDataLocation` gives (without the "/SMuFL/Fonts")
#ifdef Q_OS_WIN
      // take only the first two entries of that list on Windows (on Mac it is 2 elements only anyway)
      QStringList systemFontsPaths = QStandardPaths::standardLocations(QStandardPaths::GenericDataLocation).mid(0, 2);
#else
      QStringList systemFontsPaths = QStandardPaths::standardLocations(QStandardPaths::GenericDataLocation);
#endif
      for (QString& systemFontsPath : systemFontsPaths) {
            systemFontsPath += "/SMuFL/Fonts";
            scanUserFonts(systemFontsPath, true);
            }
```

Reply:
```
> echo $env:CommonProgramFiles
C:\Program Files\Common Files
```

Same behaviour when entering %COMMONPROGRAMFILES% into Windows Explorer...🤔

Reply:
Here too... so Qt's docs seem wrong too, as fonts I have in "%PROGRAMDATE%/SMuFL/Fonts" are getting loaded !ß1

Reply:
```
Debug: Found 18 user score font s  in "C:/Users/Jojo/Documents/MuseScore3/Score Fonts" . (C:\Users\Jojo\Documents\GitHub\MuseScore\libmscore\sym.cpp:6653, static void Ms::ScoreFont::scanUserFonts(const QString&, bool))
Debug: Found 1 system score font   in "C:/Users/Jojo/AppData/Local/SMuFL/Fonts" . (C:\Users\Jojo\Documents\GitHub\MuseScore\libmscore\sym.cpp:6653, static void Ms::ScoreFont::scanUserFonts(const QString&, bool))
Debug: Found 1 system score font   in "C:/ProgramData/SMuFL/Fonts" . (C:\Users\Jojo\Documents\GitHub\MuseScore\libmscore\sym.cpp:6653, static void Ms::ScoreFont::scanUserFonts(const QString&, bool))
``

Reply:
Well, the [Qt docs](https://doc.qt.io/qt-6/qstandardpaths.html#:~:text=%22C:/ProgramData%22) seem to be consistent with what `QStandardPaths::standardLocations` is outputting. But it's just not the same as the SMuFL is prescribing.

Reply:
Yes, Qt seems correct, SMuFL wrong.
Hmm, maybe not:  my Mu3 finds them there because I told it to look there?
I don't have any other software to test this...

Reply:
I can confirm that the [MakeMusic Fonts Installer](https://makemusic.zendesk.com/hc/en-us/articles/1500013053461-MakeMusic-Fonts-and-Licensing-Information) installs the fonts in `C:\Program Files\Common Files\SMuFL\Fonts`.

...or well, it installs the metadata.json files there and installs the actual font files into `C:\Windows\Fonts` 🫤

Reply:
Great. So a bug to fix in my implementation ;-).
Can't we use that env-var though, rather than a hard wired path?

Reply:
Maybe
```cpp
      systemFontsPaths[1] = getenv("%COMMONPROGRAMFILES%"); // "C:/Program Files/Common Files"

```

Reply:
Yep, almost. `qgetenv("CommonProgramFiles");` works 🎉

Reply:
Wonderful!

Reply:
Linux probably also needs such a patch, since Qt supposedly returns `"~/.local/share"`, `"/usr/local/share"` and `"/usr/share"`, but we want `$XDG_DATA_DIRS` and `$XDG_DATA_HOME`.
But that seems quite annoying to implement because `$XDG_DATA_DIRS` is actually a list of directories separated by colons...

Reply:
On Windows I've appended `.replace("\\", "/")`, basically for cosmetical reasons.

Reply:
> Linux probably also needs such a patch, since Qt supposedly returns `"~/.local/share"`, `"/usr/local/share"` and `"/usr/share"`, but we want `$XDG_DATA_DIRS` and `$XDG_DATA_HOME`. But that seems quite annoying to implement because `$XDG_DATA_DIRS` is actually a list of directories separated by colons...

Got it working with
```c++
QStringList xdgDataDirs = QString::fromLocal8Bit(qgetenv("XDG_DATA_DIRS")).split(':');
systemFontsPaths = xdgDataDirs << qgetenv("XDG_DATA_HOME");
```